communities.json

```diff
@@ -217,6 +217,14 @@
       "inviteCode": "b4Y7jfD",
       "githubUrl": "https://github.com/RiiConnect24"
     },
+    {
+      "logo": "openshopchannel.png",
+      "title": "Open Shop Channel",
+      "quote": "The Open Shop Channel is a library of homebrew apps for the Nintendo Wii console.",
```

Comment:
The `"quote"` is intended to be about your experience with Discord, would you mind writing a short blurb about your experience using Discord?

Reply:
Updated